app/collins/models/RemoteAssetFinder.scala

```diff
@@ -279,7 +280,8 @@ object RemoteAssetFinder {
     val key = searchParams.paginationKey + clients.map { _.tag }.mkString("_")
     val stream = Cache.getAs[RemoteAssetStream](key).getOrElse(new RemoteAssetStream(clients, searchParams))
     val results = stream.slice(pageParams.page * pageParams.size, (pageParams.page + 1) * (pageParams.size))
-    Cache.set(key, stream, 30)
+    val timeout = MultiCollinsConfig.queryCacheTimeout
```

Comment:
latency and timeliness differences between sites could be a reason, but we are far enough away from that that, IMO, you should ignore this, especially if it is hard to do